docs/docs/sourcing-from-kentico-cloud.md

```diff
@@ -120,40 +120,37 @@ exports.onCreateNode = ({ node, actions: { createNodeField } }) => {
 Now that you have a pretty way to define the path for your pages, you can create the pages programmatically:
 
 ```javascript:title=gatsby-node.js
-const path = require(`path`);
-
-exports.onCreateNode ... // As above
```

Comment:
Note that I removed this as it was preventing prettier to reformat this section automatically.

Reply:
It might help to show that this code snippet is editing the same file as above by adding the code in the section above it and highlighting the new lines that are added with this section. 

E.g.

```
const path = require(`path`) // highlight-line

exports.onCreateNode = ({ node, actions: { createNodeField } }) => {
  if (node.internal.type === `KenticoCloudItemArticle`) {
    createNodeField({
      node,
      name: `slug`,
      value: node.elements.url_pattern.value,
    })
  }
}

// highlight-start
exports.createPages = async ({ graphql, actions }) => {
  const { createPage } = actions
  // Query data from Kentico
  const result = await graphql(`
    {
      allKenticoCloudItemArticle {
        edges {
          node {
            fields {
              slug
            }
          }
        }
      }
    }
  `)

  // Create pages
  result.data.allKenticoCloudItemArticle.edges.forEach(({ node }) => {
    console.log(node.fields.slug)
    createPage({
      path: node.fields.slug,
      component: path.resolve(`src/templates/article.js`),
      context: {
        slug: node.fields.slug,
      },
    })
  })
}
// highlight-end
```

The comment seems like it could be helpful so I think we could at the least just put the whole thing as a comment so prettier can still format it:

```
// exports.onCreateNode ... code looks same as above snippet
```